base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -75,6 +76,49 @@ std::string HumanFriendlyStateName(cvd::InstanceState state) {
   return name;
 }
 
+// Adds more information to the json object returned by cvd_internal_status,
+// including some that cvd_internal_status normally returns but doesn't when the
+// instance is not running.
+void EnrichInstanceJson(const selector::LocalInstanceGroup& group,
+                        const cvd::Instance& instance,
+                        Json::Value& instance_json) {
+  CHECK(instance_json.isMember("instance_name") &&
+        instance_json["instance_name"].isString())
+      << "JSON returned from status fetcher has unexpected format";
```

Comment:
Should this function return a `Result` instead rather than `abort`ing based on invalid output from a different executable?

Reply:
This check shouldn't be here, the code below doesn't even access that member from the JSON object.